### PR TITLE
Refactor checkbox tags

### PR DESCRIPTION
**Pull request by kammon10**
#### What's this PR do?
refactor checkbox html

#### Where should the reviewer start?
index.html

#### How should this be manually tested?
na

#### Any background context you want to provide?
needed to access boxes for search

#### What are the relevant tickets?
none

#### Screenshots (if appropriate)
none

#### Questions:
please look over

### CODE REVIEW

**Review by cagallo — APPROVED** [review]
Good job! Think we might be able to use a single class, but we will see!

File: src/index.html [hunk 1]
```diff
@@ -27,80 +27,99 @@ <h1>What's Cookin'</h1>
         <h2>Filter</h2>
         <div class="filters">
           <div class="single-filter">
-            <input type="checkbox" name="antipasti">
-            <label for="antipasti">Antipasti</label>
+            <label for="antipasti">
```
Comment (anewb87) [inline src/index.html:30]: Doing some research- we think/hope this is going to help us grab the values we need easily so we can manipulate the DOM.

File: src/index.html [hunk 2]
```diff
@@ -27,80 +27,99 @@ <h1>What's Cookin'</h1>
...
+              <input type="checkbox" id="antipasti" name="antipasti" value="yes">Antipasti
+            </label>
           </div>
           <div class="single-filter">
-            <input type="checkbox" name="antipasto">
-            <label for="antipasto">Antipasto</label>
+            <label for="antipasto">
```
Comment (anewb87) [inline src/index.html:35]: lather. rinse. repeat.

**Review by anewb87 — COMMENTED** [review]
Hopefully this will help us grab info easily.

File: src/index.html (hunk 1 above)
Comment (aliroemhildt) [reply src/index.html:30]: this looks like it's heading in the right direction! I remember using the label for radio buttons a few projects ago, and followed the same setup where the label was the same as what was listed for that button (aka 'antipasti' for 'Antipasti')

**Pull request merged**